### ISSUE TITLE
Basque translation

### ISSUE THREAD

**Issue by Porrumentzio** [squidfunk/mkdocs-material#5921]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "eu",
  "direction": "ltr",
  "action.edit": "Editatu orri hau",
  "action.skip": "Joan zuzenean edukira",
  "action.view": "Ikusi orri honen iturburua",
  "announce.dismiss": "Ez erakutsi hau berriro",
  "blog.archive": "Artxiboa",
  "blog.categories": "Kategoriak",
  "blog.categories.in": "kategoria",
  "blog.continue": "Jarraitu irakurtzen",
  "blog.draft": "Zirriborroa",
  "blog.index": "Itzuli aurkibidera",
  "blog.meta": "Metadatuak",
  "blog.references": "Erlazionatutako estekak",
  "clipboard.copy": "Kopiatu arbelean",
  "clipboard.copied": "Arbelean kopiatuta",
  "consent.accept": "Onartu",
  "consent.manage": "Kudeatu ezarpenak",
  "consent.reject": "Ukatu",
  "footer": "Orri-oina",
  "footer.next": "Hurrengoa",
  "footer.previous": "Aurrekoa",
  "header": "Atalburua",
  "meta.comments": "Iruzkinak",
  "meta.source": "Iturburua",
  "nav": "Nabigazioa",
  "readtime.one": "Minutu batean irakurtzeko",
  "readtime.other": "# minututan irakurtzeko",
  "rss.created": "RSS jarioa",
  "rss.updated": "Eduki eguneratuen RSS jarioa",
  "search": "Bilatu",
  "search.placeholder": "Bilatu",
  "search.share": "Partekatu",
  "search.reset": "Garbitu",
  "search.result.initializer": "Bilaketa hasieratzen",
  "search.result.placeholder": "Idatzi bilatzen hasteko",
  "search.result.none": "Bat datorren dokumenturik ez",
  "search.result.one": "Bat datorren dokumentu bat",
  "search.result.other": "Bat datozen # dokumentu",
  "search.result.more.one": "Bat gehiago orri honetan",
  "search.result.more.other": "# gehiago orri honetan",
  "search.result.term.missing": "Falta da",
  "select.language": "Hautatu hizkuntza",
  "select.version": "Hautatu bertsioa",
  "source": "Joan biltegira",
  "source.file.contributors": "Kolaboratzaileak",
  "source.file.date.created": "Sortze data",
  "source.file.date.updated": "Azken eguneratzea",
  "tabs": "Fitxak",
  "toc": "Edukiak",
  "top": "Igo goraino"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by Porrumentzio** [comment]
Hope that works!
By the way, you should internationalize whole sentences, not single words like `in`, because in many languages this type of words (prepositions) come after the substantive (they are in fact postpositions). The right way to internationalize sentences would be including all the elements with parameters, e.g.:
- `in {category}`, so we can translate it as `{category}-n`
Also, not all languages manage grammatical number in the same way. That means that not all languages differentiate singular (1) and plural (>1), but that some languages differentiate 1, numbers ending in -5 and all other numbers, to give an example. For those languages this internationalization (singular and plural) doesn't work. Read further at the end of [this chapter](https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html)

**Comment by squidfunk** [comment]
Thanks for the PR and the feedback! You're right in that our translation system needs some improvement, and we're aware of the situation. However, there are some technical challenges that we first need to solve before we can start switching to a better approach. We'll revisit this problem in the future!

By the way, happy to see Basque here! Big fan of San Sebastian ❤️

**Comment by Porrumentzio** [comment]
If you need any help, theoretical or practical, I'm willing to help, so you could ask me

Thanks!

2023(e)ko abuztuaren 30(a) 18:09:46 (CEST)-(e)an, Martin Donath ***@***.***>-(e)k hau idatzi zuen:

**Comment by squidfunk** [comment]
Released as part of [9.2.6](https://squidfunk.github.io/mkdocs-material/changelog/#9.2.6).
